<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="MainMenuPage">
     <property name="styleSheet">
      <string notr="true">QWidget { background-color: rgb(120, 0, 22) }</string>
     </property>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(255, 47, 68); color: white; }</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="infinityButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(255, 127, 0); color: white; }</string>
      </property>
      <property name="text">
       <string>INFINITY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(255, 47, 68); color: white; }</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color: white; }</string>
      </property>
      <property name="text">
       <string>Zombie Rising</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StartPage">
     <property name="styleSheet">
      <string notr="true">QWidget { background-color: rgb(120, 0, 22) }</string>
     </property>
     <widget class="QLabel" name="startTitleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>761</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color: white; }</string>
      </property>
      <property name="text">
       <string>Survive 3 levels to beat the game</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="startGameButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>300</y>
        <width>220</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(50, 200, 50); color: white; border-radius: 10px; }
QPushButton:hover { background-color: rgb(70, 220, 70); }</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backToMenuButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>220</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(200, 50, 50); color: white; border-radius: 10px; }
QPushButton:hover { background-color: rgb(220, 70, 70); }</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GamePage">
     <property name="styleSheet">
      <string notr="true">QWidget { background-color: rgb(120, 120, 120) }</string>
     </property>
     <widget class="GameView" name="gameArea" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>751</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { border: 3px solid black; }</string>
      </property>
     </widget>
     <widget class="QLabel" name="slot0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/heal.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="slot1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>480</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/shield.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 53, 53);
color: rgb(225, 0, 4);</string>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QLabel" name="levelLabel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Level 1</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="healthBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar { background-color: gray; }
QProgressBar::chunk { background-color: rgb(16, 244, 0); }</string>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="killsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Kills: 0/30</string>
      </property>
     </widget>
     <widget class="QLabel" name="scoreLabel">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>500</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="slot2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/powerup.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PauseMenuPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 120, 120);</string>
     </property>
     <widget class="QLabel" name="gamePausedLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Game Paused</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resumeButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(235, 0, 0);
background-color: rgb(72, 72, 72);</string>
      </property>
      <property name="text">
       <string>RESUME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainMenuPauseButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(235, 0, 0);
background-color: rgb(72, 72, 72);</string>
      </property>
      <property name="text">
       <string>MAIN MENU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitPauseButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(235, 0, 0);
background-color: rgb(72, 72, 72);</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="pauseScoreLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Score: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Level2Page">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(113, 113, 113);</string>
     </property>
     <widget class="QLabel" name="level2Label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Level Up! Level 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="weaponImageLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>421</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="weaponNameLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Weapon Unlocked: The Blaster</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continueButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 61, 61);
color: rgb(236, 0, 4);</string>
      </property>
      <property name="text">
       <string>CONTINUE</string>
      </property>
     </widget>
     <zorder>weaponImageLabel</zorder>
     <zorder>level2Label</zorder>
     <zorder>weaponNameLabel</zorder>
     <zorder>continueButton</zorder>
    </widget>
    <widget class="QWidget" name="youWonPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 100, 0);</string>
     </property>
     <widget class="QLabel" name="youWonLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>400</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>You Won!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="restartLevelsButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>280</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(50, 200, 50); color: white; border-radius: 10px; }
QPushButton:hover { background-color: rgb(70, 220, 70); }</string>
      </property>
      <property name="text">
       <string>Restart Levels</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainMenuWonButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>330</y>
        <width>280</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(100, 100, 200); color: white; border-radius: 10px; }
QPushButton:hover { background-color: rgb(120, 120, 220); }</string>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitGameButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>410</y>
        <width>280</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(200, 50, 50); color: white; border-radius: 10px; }
QPushButton:hover { background-color: rgb(220, 70, 70); }</string>
      </property>
      <property name="text">
       <string>Exit Game</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="YouLostPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 127, 127);</string>
     </property>
     <widget class="QPushButton" name="restartButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(232, 0, 0);
background-color: rgb(70, 70, 70);</string>
      </property>
      <property name="text">
       <string>RESTART</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>You Lost!</string>
      </property>
     </widget>
     <widget class="QLabel" name="finalScoreLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>351</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Final Score: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameView</class>
   <extends>QWidget</extends>
   <header>gameview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
